### PR TITLE
Query files

### PR DESCRIPTION
**Pull request by I-am-Scott-White**
Aggregate wrap added to all aggregate text files.

### CODE REVIEW

**Review by mdjohns — CHANGES_REQUESTED** [review]
Good news: We've got the queries we need for SO Scores by Level, SO Scores by Instructor, and SO Scores by Course. :tada: 

But...
I'm still not sure we have what we need for SO Scores by Student. There is a query for student overall SO scores for a semester (`QueryStudentOverallAverage`), but that's scoped to the whole semester and we need something scoped to a course instance since that will be the page it's displayed on. Example: Given a course instance, like Programming I in Fall 2020, we need to return each student and their average overall SO score. How hard would it be to get that data? 

Also, I'm not sure about Scores by Term either. I've been fighting with the mental concept of that one for weeks, so I might need some more guidance from @jcypret here.

File: seeders/users.seeder.js [hunk 1]
```diff
@@ -1,48 +1,34 @@
 import { Seeder } from "mongoose-data-seed";
 import User from "../src/models/User";
-import { hashPassword, createExpirationDate } from "../src/utils/auth";
+import Instructor from "../src/models/Instructor";
+import { hashPassword } from "../src/utils/auth";
 
-const data = [
```
[the remark this answers is not in the document]
Comment (I-am-Scott-White) [reply seeders/users.seeder.js:5]: This issue should be resolved.

File: queries/QueryByCourse [hunk 2]
```diff
@@ -0,0 +1,143 @@
+// Returns mean overall SO scores for each course by specified term
+
+dcia.courseinstances.aggregate([
```
[the remark this answers is not in the document]
Comment (I-am-Scott-White) [reply queries/QueryByCourse]: This issue is resolved

**Comment by mdjohns** [comment]
> line 97 'studentWorkProject' should be 'studentOutcome'. I think I have been trying to push to fast today.
re-added in 6ed7d03

**Pull request merged**